src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/linalg/FourierLinearOperator.java

```diff
@@ -0,0 +1,99 @@
+package org.broadinstitute.hellbender.tools.coveragemodel.linalg;
```

Comment:
As we discussed, these classes should probably be moved to the hellbender.utils package so they can be more visible and generally used.